test/ArgumentForwardingTests/ArgumentForwardingTests.cs

```diff
@@ -262,8 +263,8 @@ private string[] RawEvaluateArgumentString(string testUserArgument)
             {
                 StartInfo = new ProcessStartInfo
                 {
-                    FileName = ReflectorPath,
-                    Arguments = testUserArgument,
+                    FileName = Env.GetCommandPath("dotnet", ".exe", ""),
+                    Arguments = $"{ReflectorPath} {testUserArgument}",
```

Comment:
It's unfortunately more complicated than that to fix this in all cases (which is exactly what this is testing :smile: )

But that should at least not break in the simple case.

There are other places where we have this issue as well in our build, so I wouldn't block on it.